drivers/adc/ad9250/ad9250.c

```diff
@@ -516,23 +487,17 @@ int32_t ad9250_test_mode(struct ad9250_dev *dev,
  * @param dev - The device structure.
  * @param adj - The offset adjust value in LSBs from +31 to -32.
  *
- * @return Returns negative error code or the set offset adjustment.
+ * @return Returns negative error code or 0 in case of success.
  *******************************************************************************/
 int32_t ad9250_offset_adj(struct ad9250_dev *dev,
 			  int32_t adj)
 {
-	int32_t ret = 0;
-
-	if((adj >= -32) && (adj <= 31)) {
-		ret = ad9250_write(dev,
-				   AD9250_REG_OFFSET,
-				   AD9250_REG_OFFSET_ADJUST(adj));
-	} else {
-		return (ad9250_read(dev,
-				    AD9250_REG_OFFSET));
-	}
-
-	return ret;
+	if((adj >= -32) && (adj <= 31))
```

Comment:
This change is a little bit suspicios because it changes more than what this commit is about.
The commit should be only about addint the extra parameter to the ad9250_read.
I'm also not even sure if this is functionally correct, removing the ad9250_read call from that else branch.

Reply:
It is stated in the commit message:
`Until now, the function was returning the value read, while the checks
through the entire driver were expecting a return code.
(SUCCESS/FAILURE).
`